### PR TITLE
add gatsby main off threads

### PR DESCRIPTION
**Pull request by eduard-deriv**
Changes:

-  changed the script loading strategy. There is almost no information on this experimental technology on the Internet, perhaps I added the strategy incorrectly.

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ X ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: gatsby-browser.js [hunk 1]
```diff
@@ -137,6 +137,7 @@ export const onClientEntry = () => {
         addScript({
```
Comment (habib-deriv) [inline gatsby-browser.js:137]: Actually we are not using this GTM
Our current GTM container ID, GTM-NF7884S


We can cleanup the old gtm

File: gatsby-browser.js [hunk 2]
```diff
@@ -145,12 +146,14 @@ export const onClientEntry = () => {
                 j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
                 'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
                 })(window,document,'script','dataLayer','GTM-TNX2ZKH');`,
```
Comment (habib-deriv) [inline gatsby-browser.js:148]: Same here.. Looks like this GTM id is outdated. Please speak with BI team about this dataLayer, which is not firing

File: gatsby-config.js [hunk 3]
```diff
@@ -18,6 +18,13 @@ module.exports = {
         author: 'Deriv.com',
         siteUrl: site_url,
     },
+    partytownProxiedURLs: [
+        `https://www.googletagmanager.com/gtm.js?id=GTM-TNX2ZKH`,
```
Comment (habib-deriv) [inline gatsby-config.js:22]: Same, not using this

**Review by habib-deriv — APPROVED** [review]
LGTM

**Review by ali-hosseini-fs — APPROVED** [review]

**Review by prince-deriv — APPROVED** [review]

**Pull request merged**